components/sidebar/sidebar.tsx

```diff
@@ -140,13 +177,17 @@ export default class Sidebar extends React.PureComponent<Props, State> {
                         <AddChannelDropdown
                             showNewChannelModal={this.showNewChannelModal}
                             showMoreChannelsModal={this.showMoreChannelsModal}
+                            showCreateCategoryModal={this.showCreateCategoryModal}
                             canCreateChannel={this.props.canCreatePrivateChannel || this.props.canCreatePublicChannel}
                             canJoinPublicChannel={this.props.canJoinPublicChannel}
                         />
                     </div>
                 </div>
                 <Pluggable pluggableName='LeftSidebarHeader'/>
-                <SidebarCategoryList handleOpenMoreDirectChannelsModal={this.handleOpenMoreDirectChannelsModal}/>
+                <SidebarCategoryList
+                    handleOpenMoreDirectChannelsModal={this.handleOpenMoreDirectChannelsModal}
+                    newCategoryIds={this.newCategoryIds}
```

Comment:
`this.newCategoryIds` affects rendering, so it should go in `this.state.newCategoryIds`